perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -813,13 +813,19 @@ sub _get_current_service_task {
     @$servicetasks = grep { $_->{ServiceTaskSchedules} } @$servicetasks;
     return unless @$servicetasks;
 
+    my $service_name = service_name_override()->{$service->{ServiceId}} || $service->{ServiceName};
+    my $today = DateTime->now->set_time_zone(FixMyStreet->local_time_zone);
     my ($current, $last_date);
-    foreach ( @$servicetasks ) {
-        my $end = construct_bin_date($_->{EndDate});
-
-        next if $last_date && $end && $end < $last_date;
-        $last_date = $end;
-        $current = $_;
+    foreach my $task ( @$servicetasks ) {
```

Comment:
Not for now, but feels like we could take the result of the GetServiceUnits API call and filter out blank/out-dated service tasks first off and then not have to call this function twice in `bin_services_for_address`. But it's only another loop through the same data.